CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+
+### New feature support
+*
+
+### Added
+*
+
+### Removed
+*
+
```

Comment:
We need a `## [1.2.1]` tag and indicate in the changelog that we revendored using the 1.7.0 version of the gem.  We have not done this before but it would probably be good to add a link to node_utils changelog https://github.com/cisco/cisco-network-node-utils/blob/develop/CHANGELOG.md#v170